Makefile

```diff
@@ -747,6 +747,7 @@ openstack_init: openstack_wait
 	bash -c 'test -f ${OPERATOR_BASE_DIR}/openstack-operator/config/samples/operator_v1beta1_openstack.yaml || make openstack_repo'
 	oc apply -f ${OPERATOR_BASE_DIR}/openstack-operator/config/samples/operator_v1beta1_openstack.yaml
 	oc wait openstack/openstack -n ${OPERATOR_NAMESPACE} --for condition=Ready --timeout=${TIMEOUT}
+	timeout ${TIMEOUT} bash -c "while ! (oc get mutatingwebhookconfiguration | grep -E '^(openstack|infra)-operator-mutating-webhook-configuration' | wc -l | grep -q -e 2); do sleep 1; done"
```

Comment:
This can happen for the OpenStack operator webhooks too, so I think we would want to cover that case here as well.  Soon OpenStack Baremetal operator webhooks will be present as well, and this would need to be adjusted.  We are working on a more comprehensive solution, but fixes like these make sense in the meantime.

Reply:
Sorry, I misread your code.  I see that OpenStack operator is already included.  The problem, though, isn't the `MutatingWebhookConfiguration` resources themselves.  It's that the associated K8S `Service` hasn't been created yet.  That is what really needs to be examined here.

Reply:
Right, it will work and be more simple too, let me update, thanks!